internal/cluster/cluster.go

```diff
@@ -1039,6 +1040,88 @@ func (m *Manager) IsReadyForInstallation(c *common.Cluster) (bool, string) {
 	return true, ""
 }
 
+func (m *Manager) detectAndStoreCollidingIPsForCluster(cluster *common.Cluster, db *gorm.DB) error {
+	if db == nil {
+		db = m.db
+	}
+	// We want to calculate ip collisions only when in pre-install states since it is needed for pre-install validations
+	allowedStates := []string{
+		models.ClusterStatusPendingForInput,
+		models.ClusterStatusInsufficient,
+
```

Comment:
redundant space. Plus do we want to skip pending-for-input state because it may take quite some time?